cfme/rest/gen_data.py

```diff
@@ -557,23 +557,25 @@ def blueprints(request, rest_api, num=2):
     return _creating_skeleton(request, rest_api, 'blueprints', data)
 
 
-def conditions(request, rest_api, num=2):
+def conditions(appliance, request, num=2):
```

Comment:
REQUIRED: I wouldn't change this. your change looks quite reasonable. however, you change only two procedures in this file whereas the rest is left the same. code should look similar.

Reply:
Hi @izapolsk, I don't understand what you're saying. Before making the above changes, I had tried to access appliance version with `rest_api` argument, but didn't find a way to do so. And besides `conditions` and `policies` aren't used at any place other than `test_rest_control.py`, so I assumed it was safe to make those changes.

Reply:
@valaparthvi
1. please expand this file and look thru all methods here. all of them except modified by you expect you to pass rest_api and only couple of methods demand appliance. So, you put in difference in method signatures.

2. when it comes to version, you can use two ways. 
2.1. if you look at appliance.version , you will see that it was retrieved thru rest_api and wrapped into Version(). So, you can use the same approach here. 
2.2 you can replace rest_api calls with appliance.rest_api in whole module and update tests respectively.

I would choose second way. it would make code more neat.